src/Makefile.am

```diff
@@ -124,6 +126,7 @@ INCLUDES =                                   \
 	${GUILE_CFLAGS}
 
 gnotime_LDADD =          \
+	-lm \
 	${GLIB_LIBS}     \
```

Comment:
You want `-lm` to be last, not first. The linker-loader resolves symbols in order.

Reply:
Thank you for the hint, I'm not very knowledgable in this realm and will fix this accordingly.